Quaver.Shared/Database/Maps/Mapset.cs

```diff
@@ -89,7 +89,10 @@ public string ExportToZip(bool openInExplorer = true)
                                 Logger.Debug($"Successfully converted osu beatmap: {osuPath}", LogType.Runtime);
                                 break;
                             case MapGame.Etterna:
-                                var stepFile = new StepFile(map.Path);
+                                var stepFile = new StepFile(map.Path)
+                                {
+                                    Background = Path.GetFileName(map.BackgroundPath)
+                                };
```

Comment:
What exactly is this portion intended to be doing? Isn't the background already set during the conversion?

Doing this will also break existing leaderboards for these files when exported and re-imported.

Reply:
It is, but it don't work for specific reason, please look this video (i will timestamps and explain what's going on) : https://www.youtube.com/watch?v=ogfbq6LCbzg

On current build (0.26.2) : 
0:07 : Background is present in menu.
0:11 : Background is present in game.
0:17 : Export chart.
0:22 : import .qp file.
0:55 : **Background not present in menu after importing the .qp file that got exported.**
0:59 : **Background not in game either.**

On build with this PR :
1:17 : Detect and load DB.
1:56 : Don't have anything to do with this build, osu bg not loading, that's cause i'm using a build were osu.db path didn't get fixed yet.
2:00 : **Banner correctly loaded.**
2:06 : Show the background in game.
2:14 : Export chart.
2:19 : import .qp file.
2:52 : **Banner present even after importing the .qp file.**
2:57 : **Background present in game.**

As to why that "bug" happen in the first place : 
3:15 : background is named bg.
3:22 : background is named differently in the file.

So, what happen is when exporting the .qp file, when creating a "new StepFile" in the code, the background path is set like it's named in the file "aka 2minutesinphotoshop", AND bg is actually imported correctly because of the "copy over background if necessary" just after, which work cause the path is correct there. 

To fix that, i set the background path directly from the mapset.cs, making it sure the background will be correctly named after the background file that will get imported.

Reply:
I'm gonna remove this portion for now (the rest of the PR is fine). I need to think of a proper way to fix this. It has other side effects which makes this more complex issue than just the background not being set.